RELEASE.md

```diff
@@ -0,0 +1,53 @@
+# Boxo Releases
+
+## Release Policies
+### Versioning
+Boxo only releases minor and patch versions, and has no plans to change major versions from v0.
+
+Minor versions contain new or removed functionality, or significant changes in dependencies.
+
+Patch versions contain "fixes", which are generally small, targeted, and involve limited changes in functionality (such as bug fixes, security fixes, etc.). The amount of backporting depends on the severity of the issue and the impact on users...most non-critical fixes won't be backported.
```

Comment:
```suggestion
Patch versions contain "fixes", which are generally small, targeted, and involve limited changes in functionality (such as bug fixes, security fixes, etc.). The amount of backporting depends on the severity of the issue and the impact on users.  Most non-critical fixes won't be backported.
```

Reply:
Backporting is also sort of similar in nature to the topic of handling deprecation that I was adding in https://github.com/ipfs/boxo/pull/260 .  I think we probably want to have those things sit close to each other as they are all things around setting user expectations.

Reply:
Agree maybe we should have this section in the main README?

Reply:
I think the "release policies" should all live in a single place.  This could be the readme or or in this document (and readme can link to "release policies").

We want to have headings for 
Backwards compatibility
Backporting